docs/REFERENCE.md

```diff
@@ -123,39 +165,89 @@ const errResult = Err.of(new UserNotFoundError());
 
 ### err.map(callback)
 
-Do nothing for `Err` results.
-Interface common for both types of results: [result.map(callback)](#resultmapcallback)
+No operation is performed for `Err` results. This interface is common for both `Ok` and `Err` results: [result.map(callback)](#resultmapcallback)
+
+**Examples:**
+
+```ts
+import { Ok } from 'type-safe-errors';
+import { UserNotFoundError } from './errors';
+
+const errResult = Err.of(new UserNotFoundError());
+// No operation is performed, the original Err result is returned
+const sameErrResult = errResult.map(err => 123); 
+```
 
 ---
 
 ### err.mapErr(ErrorClass, callback)
 
-Map `Err` result of specified class to a different result.
+Map the `Err` result of specified class to a different result.
```

Comment:
```suggestion
Map the `Err` result of specified class to a different result. If you don't explicitly return value of `Err` type it will be mapped to `Ok`
```